test/inductor/test_minifier.py

```diff
@@ -149,6 +151,7 @@ def inner(x):
         self.assertEqual(test_proc.returncode, repro_proc.returncode)
         self.assertNotEqual(test_proc.returncode, 0)
 
+    @unittest.skipIf(not _HAS_TRITON, "openAI Triton unavailable")
     def test_after_aot_cpu_runtime_error(self):
         self._test_after_aot_runtime_error("cpu", CPP_RUNTIME_ERROR)
```

Comment:
Why do these cpu tests require triton?

Reply:
w/o this flag we get the following errors:
```
FAIL: test_after_aot_cpu_accuracy_backend_passes (__main__.MinifierTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "inductor/test_minifier.py", line 191, in test_after_aot_cpu_accuracy_backend_passes
    self._test_after_aot_backend_passes("cpu", 4, CPP_ACCURACY_ERROR)
  File "inductor/test_minifier.py", line 181, in _test_after_aot_backend_passes
    self.assertEqual(proc.returncode, 0)
  File "/home/riship/.local/lib/python3.8/site-packages/torch/testing/_internal/common_utils.py", line 2859, in assertEqual
    assert_equal(
  File "/home/riship/.local/lib/python3.8/site-packages/torch/testing/_comparison.py", line 1270, in assert_equal
    raise error_metas[0].to_error(msg)
AssertionError: Scalars are not equal!

Absolute difference: 1
Relative difference: inf

======================================================================
FAIL: test_after_aot_cpu_accuracy_error (__main__.MinifierTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "inductor/test_minifier.py", line 108, in test_after_aot_cpu_accuracy_error
    (tb1, tb2), _ = self._test_after_aot("cpu", CPP_ACCURACY_ERROR, 4)
  File "inductor/test_minifier.py", line 94, in _test_after_aot
    (test_proc, _, repro_proc), _ = self._run_full_test(
  File "/home/riship/.local/lib/python3.8/site-packages/torch/_dynamo/test_minifier_common.py", line 126, in _run_full_test
    self.assertIsNotNone(repro_dir)
AssertionError: unexpectedly None

======================================================================
FAIL: test_after_aot_cpu_compile_backend_passes (__main__.MinifierTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "inductor/test_minifier.py", line 185, in test_after_aot_cpu_compile_backend_passes
    self._test_after_aot_backend_passes("cpu", 2, CPP_COMPILE_ERROR)
  File "inductor/test_minifier.py", line 181, in _test_after_aot_backend_passes
    self.assertEqual(proc.returncode, 0)
  File "/home/riship/.local/lib/python3.8/site-packages/torch/testing/_internal/common_utils.py", line 2859, in assertEqual
    assert_equal(
  File "/home/riship/.local/lib/python3.8/site-packages/torch/testing/_comparison.py", line 1270, in assert_equal
    raise error_metas[0].to_error(msg)
AssertionError: Scalars are not equal!

Absolute difference: 1
Relative difference: inf

======================================================================
FAIL: test_after_aot_cpu_compile_error (__main__.MinifierTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "inductor/test_minifier.py", line 103, in test_after_aot_cpu_compile_error
    (tb1, tb2), _ = self._test_after_aot("cpu", CPP_COMPILE_ERROR, 2)
  File "inductor/test_minifier.py", line 94, in _test_after_aot
    (test_proc, _, repro_proc), _ = self._run_full_test(
  File "/home/riship/.local/lib/python3.8/site-packages/torch/_dynamo/test_minifier_common.py", line 126, in _run_full_test
    self.assertIsNotNone(repro_dir)
AssertionError: unexpectedly None
======================================================================
FAIL: test_after_aot_cpu_runtime_backend_passes (__main__.MinifierTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "inductor/test_minifier.py", line 188, in test_after_aot_cpu_runtime_backend_passes
    self._test_after_aot_backend_passes("cpu", 2, CPP_RUNTIME_ERROR)
  File "inductor/test_minifier.py", line 181, in _test_after_aot_backend_passes
    self.assertEqual(proc.returncode, 0)
  File "/home/riship/.local/lib/python3.8/site-packages/torch/testing/_internal/common_utils.py", line 2859, in assertEqual
    assert_equal(
  File "/home/riship/.local/lib/python3.8/site-packages/torch/testing/_comparison.py", line 1270, in assert_equal
    raise error_metas[0].to_error(msg)
AssertionError: Scalars are not equal!

Absolute difference: 1
Relative difference: inf

======================================================================
FAIL: test_after_aot_cpu_runtime_error (__main__.MinifierTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "inductor/test_minifier.py", line 153, in test_after_aot_cpu_runtime_error
    self._test_after_aot_runtime_error("cpu", CPP_RUNTIME_ERROR)
  File "inductor/test_minifier.py", line 143, in _test_after_aot_runtime_error
    (test_proc, _, repro_proc), _ = self._run_full_test(
  File "/home/riship/.local/lib/python3.8/site-packages/torch/_dynamo/test_minifier_common.py", line 126, in _run_full_test
    self.assertIsNotNone(repro_dir)
AssertionError: unexpectedly None

----------------------------------------------------------------------
Ran 12 tests in 37.946s

FAILED (failures=6, skipped=6)
```

Reply:
If it was requiring triton I would expect an import error. You should just skip on jetson.